### PR TITLE
Refactored Bifrost.MongoDb for newest MongoDb driver.

### PR DESCRIPTION
**Pull request by svethors**

### CODE REVIEW

File: Source/Bifrost.MongoDb/Concepts/ConceptSerializer.cs [hunk 1]
```diff
@@ -10,82 +10,160 @@
...
+			else if (valueType == typeof(string))
+				value = bsonReader.ReadString();
+			else if (valueType == typeof(decimal))
+				value = decimal.Parse(bsonReader.ReadString());
+
+			var concept = ConceptFactory.CreateConceptInstance(actualType, value);
+			return concept;
+		}
+
+		//public object Deserialize(BsonReader bsonReader, Type nominalType, Type actualType, IBsonSerializationOptions options)
```
Comment (einari) [inline Source/Bifrost.MongoDb/Concepts/ConceptSerializer.cs]: Please take out the commented code

**Comment by einari** [comment]
Looks good. The only thing was the commented code that shouldn't be there.

**Comment by svethors** [comment]
Of course that should not be there. Gone now.

**Comment by einari** [comment]
Thanks for the PR

**Comment by einari** [comment]
FYI: Up on MyGet now : https://www.myget.org/gallery/bifrost

**Pull request merged**